### ISSUE TITLE
Getting an issue with Spigot 1.8.8

### ISSUE THREAD

**Issue by bootthanoo** [MassiveCraft/MassiveCore#239]
![screen shot 2015-10-22 at 8 01 53 pm](https://cloud.githubusercontent.com/assets/664991/10682179/cbead2d8-78f7-11e5-86de-954367befbdd.png)
I'm running spigot 1.8.8 on Mac OSX. 

MassiveCore and Vampire doesn't work at all and MassiveCore is throwing this error.

**Comment by magnusulf** [comment]
Did you get this MassiveCore version from spigotmc.org?

**Comment by bootthanoo** [comment]
Thank you so much for replying within the day :D :D :D 

I got the plugin from http://dev.bukkit.org/bukkit-plugins/mcore/

**Comment by ulumulu1510** [comment]
The plugins on bukkit and spigot /should/ be the same. Would youpelase state what version of Java you are running?
Also please try a clean install on a freshly setup spigot server with just MassiveCore and see if the errors persist.

If all of this doesn't work, please download MassiveCore from Spigot and retry.

**Comment by bootthanoo** [comment]
I'm running

java version "1.6.0_65"

Java(TM) SE Runtime Environment (build 1.6.0_65-b14-468-11M4833)

Java HotSpot(TM) 64-Bit Server VM (build 20.65-b04-468, mixed mode)


That screenshot was from a clean install of Spigot and just MassiveCore

**Comment by ulumulu1510** [comment]
Do you have the ability to update you java version? If so I would highly recommend updating ro atleast java 1.7.

The issue says the class was not found, though obviously it is present. This can sometimes be cause by too low java versions, although we try our best to stay 1.6 compatible.

TL;DR, if you can, update java and retry please.

**Comment by bootthanoo** [comment]
Thank you :) Updating JRE and JDK to the 8 has fixed it. Thank you very
much :D
